lib/acx/horde/horde_registry.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Acx.Horde.HordeRegistry do
```

Comment:
If you change the name back in the enforcer_sever.ex would this still be needed?